scipy-tutorial/06_xarray_and_dask.ipynb

```diff
@@ -17,9 +17,58 @@
   },
```

Comment:
<p>Since this is intentional, you can insert the <code>expectexception</code>  magic in this cell. See <a href="https://xarray-contrib.github.io/xarray-tutorial/scipy-tutorial/03_computation_with_xarray.html?highlight=expectexception" target="_blank">here</a> for an example of how this is done.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>